packages/bezier-react/src/components/Emoji/Emoji.module.scss

```
@@ -0,0 +1,32 @@
+@use '../../styles/mixins/dimension';
+
+$emoji-sizes: '16', '20', '24', '30', '36', '42', '48', '60', '72', '90', '120';
+
+@mixin emoji-background($emoji-size) {
+  background-image: var(--b-emoji-background-image);
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: $emoji-size $emoji-size;
+  background-clip: content-box;
+}
+
+@layer components {
+  .Emoji {
+    --b-emoji-size: auto;
```

```suggestion
    --b-emoji-size: initial;
```

초깃값으로 더 명확해서 좋을 거 같아요. auto와 동일합니다.